packages/daily-image-api/__tests__/integration/app/query/storage/ReadDynamoDbDailyImages.test.ts

```diff
@@ -0,0 +1,34 @@
+import AWS from "aws-sdk";
+import { ReadDynamoDbDailyImages } from "../../../../../app/query/storage/ReadDynamoDbDailyImages";
+
+jest.mock("aws-sdk");
```

Comment:
Interesting! I would have written the mock to only include the method necessary for the test, but this means you have to assert the type as `any` when passing it into the constructor (which I would always prefer not to have to do) .e.g.

```
const dbMock = {
    scan: jest.fn(() => ({
        promise: jest.fn().mockResolvedValue(null)
      }));
}

const sut = new ReadDynamoDbDailyImages(dbMock as any, "tableName");
```

Reply:
Yes, I did try this, as importing + mocking an entire module seemed a bit heavyweight. However, type safety seemed a reasonable trade, and the test isn't slow either way. :smiley:

Reply:
btw, is this a request to change it to using `as any` or just an observation?

Reply:
Just an observation